### PR TITLE
APP-917: Make showkase available in PR builds

### PR DESCRIPTION
**Pull request by StylianosGakis**

### CODE REVIEW

**Review by hedvigoscar — APPROVED** [review]
lgtm

**Review by hugokallstrom — APPROVED** [review]

File: app/build.gradle.kts [hunk 1]
```diff
@@ -92,6 +93,24 @@ android {
             )
         }
 
+        named("pullrequest") {
+//            Uncomment this to test locally
+//            signingConfig = signingConfigs.getByName("debug")
+
```
Comment (hedvigoscar) [inline app/build.gradle.kts:99]:
```suggestion
```

File: app/build.gradle.kts [hunk 2]
```diff
@@ -92,6 +93,24 @@ android {
...
+            applicationIdSuffix = ".pullrequest.app"
```
Comment (hedvigoscar) [inline app/build.gradle.kts:100]:
```suggestion
            applicationIdSuffix = ".test.app"
```

**Review by hedvigoscar — APPROVED** [review]
A discussed live, let's have the `pullrequest`-config use the same config as `staging`. 🙂 In all other regards, LGTM!

**Pull request merged**